pkg/dumper/dumper.go

```diff
@@ -55,10 +57,10 @@ func NewDumper(opts ConnOpts, rdr reader.Reader) (dumper Dumper, err error) {
 	})
 
 	if dumper == nil && err == nil {
-		err = errors.New("no supported driver found")
+		err = fmt.Errorf("no supported driver found for dumper DSN %q", opts.DSN)
 	}
 
-	err = errors.Wrapf(err, "could not create dumper for dsn: '%v'", opts.DSN)
+	err = wErrors.Wrapf(err, "could not create dumper for DSN: %q", opts.DSN)
```

Comment:
Seems a little double up with the error in line 60 .... Maybe put it in an `else` or remove the DSN from line 60 error

Reply:
On line 60 it check of the driver was found (main part is `dumper == nil`) and here it wraps the error if any - the trick is that if the error is `nil` wrapped result would also be `nil` - it is not very obvious, I had to dig into the code to understand it. It worth rewriting driver search part, but here I just slightly improved error reporting as I could not understad anything from current when started to reproduce the problem and debug it.

Reply:
Added a suggestion for line 60 :smile: